documentation/contributing/pullrequests/index.markdown

```diff
@@ -134,33 +133,47 @@ If each step ends with issues, they need to be fixed before moving on.
      ```
 
-1. Update changelogs. Take advantage of `catkin_generate_changelog` command to populate new logs, then preferably edit them manually to sort out per the type of changes (e.g. bugfix, new capability, maintenance, documentation). Example of the whole command set:
+1. Update changelogs. Take advantage of `catkin_generate_changelog` command to populate new logs, then preferably edit them manually to cleanup/combine changelog messages and mark/sort them by type of change (e.g. bugfix, feature, maintenance, documentation). Have a look at previous changelogs to get an idea of the verbosity level and formatting. Example of the whole command set:
 
    ```
    cd moveit                              (Top directory of your clooned moveit repo.)
    git checkout master
    git log                                (Make sure the HEAD is what you want to release with. If it's not then update accordingly.)
    catkin_generate_changelog
    emacs `find . -iname CHANGELOG.rst`    (Edit each file. Emacs forever, but replace it if necessary :/)
+   git add `find . -iname CHANGELOG.rst`  (Stage all changelogs)
    ```
-1. Create a new tag with an appropriate version number (see the version policy section). Utilize `catkin_prepare_release` command that bumps the versions in package xml and in changelog files, creates a new tag, and pushes it to the remote repo (you can check at [github.com/ros-planning/moveit/releases](https://github.com/ros-planning/moveit/releases)). Example command:
+1. Figure out the correct next version number ([see the version policy section](#Release-versioning-policy)) and utilize the command `catkin_prepare_release` that adapts the version numbers in all package.xml and changelog files, creates a new tag, and eventually pushes to the remote repo (you can check at [github.com/ros-planning/moveit/releases](https://github.com/ros-planning/moveit/releases)). Example command:
 
    ```
-   (Assuming you're at the same directory as previously)
+   (Assuming you're at the same directory as before.)
 
-   catkin_prepare_release --bump patch    (Or without any option it suggests bumping minor version.)
-   git tag                                (Confirm among all tags that the tag with the intended version is locally created. Also go online to see the tag is uploaded.)
+   catkin_prepare_release --version x.x.x    (Replace x.x.x with the correct version number!)
    ```
 1. Run `bloom`. Open a pull request against [rosdistro](https://github.com/ros/rosdistro) as bloom suggests at the end of its run. [Example of such a request](https://github.com/ros/rosdistro/pull/13512). Example command:
 
    ```
-   bloom-release --rosdistro melodic --track melodic moveit
+   bloom-release --rosdistro melodic moveit
    ```
+1. Bump the version number to the next upcoming one and _finally_ push to the github:
+   ```
```

Comment:
I think there should be a blank line here

Reply:
Did you get weird formatting? I used vscode's life markdown preview and everything looked fine.